config/default/manager_resources_patch.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: kfserving-controller-manager
+  namespace: kfserving-system
+spec:
+  template:
+    spec:
+      containers:
+      - command:
```

Comment:
in this path we don't want to overwrite the command, image, and imagePullPolicy. Can we remove those three fields?

Reply:
removed